new.py

```diff
@@ -0,0 +1,15 @@
+import fileop
+import process
+
+class Caeser(object):
+	def __init__(self,shift):
+		self.shift = shift
+
+	def encrypt(self):
+		data = fileop.FileOperator().file_read()
+		to_file = process.Proces().processEncrypt(data,self.shift)

```

Comment:
@Nithinkrishna1 Instead of using self.shift use instance getter method to get the value of shift